analysis/src/HistoManager.cxx

```diff
@@ -3,6 +3,14 @@
 #include "TKey.h"
 #include "TClass.h"
 
+HistoManager::HistoManager() {
+    HistoManager("default");
+}
+
+HistoManager::HistoManager(const std::string& inputName) {
+    m_name = inputName;
+    event_h = new TH1D("event_h","Number of Events Processed;;Events", 21, -10.5, 10.5);
```

Comment:
Buggy if you have more than one instance of histograms that end in the same file.

Reply:
I agree, moved this to Process, and it can from there become the cut flow histogram when we add FilterProcessors.